docs-kits/kits/DCM-Kit/page_development-view.md

```diff
@@ -0,0 +1,913 @@
+---
+id: development-view
+title: Development View
+description: 'What do I have to implement?'
+sidebar_position: 3
+---
+
+![DCM kit banner](@site/static/img/DCMKitLogoIcon-min.png)
+
+This page describes the most important parts for the software implementation of the DCM standards. For the full technical specification, please refer to the standard [CX-0128 Demand and Capacity Management Data Exchange](https://catena-x.net/de/standard-library).
+
+## Introduction
+
+This document describes the `MaterialDemand`, `WeekBasedCapacityGroup`, `IdBasedRequestForUpdate` and `IdBasedComment` semantic models and the API definitions used in the DCM Catena-X network.
+
+## ASPECT MODELS
```

Comment:
Please do not use so much capital letters :)

Reply:
Fixed